ARTICLE-2.markdown

```diff
@@ -4,7 +4,11 @@
 
 A Member shall have the right to resign from the Cooperative, effective
 immediately without any action on the part of the Cooperative.
-Initial capital contribution will not be returned.
+
+Initial capital contribution will be returned immediately if the
+Collective Account (see Article 7) is deemed sufficient to support the
+return. Otherwise, initial capital contribution will be returned in
+installments such that the Collective Account can support it.
 
```

Comment:
@brodavi , what we want to add here is that the board will decide on one time settlement or deferred payments based on the situation of Collective Account, and the company plans (may be some investment might be lined up, or equity stuff going on, some legal matters, etc). The terms of deferred payment will be decided at that time only based on the circumstances